tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -621,6 +906,115 @@ def export(self, name=None):
         full_values.append(vals_)
     return array_ops.concat(full_keys, 0), array_ops.concat(full_values, 0)
 
+  def save_to_file_system(self,
+                          dirpath,
+                          mpi_size=1,
+                          mpi_rank=0,
+                          file_name_list=[],
```

Comment:
Just use None for empty, or all of the callings will share the `[]`.

Reply:
Done